index.js

```diff
@@ -17,8 +18,8 @@ module.exports = (opts, types, defaults) => {
 		try {
 			npmPath = require.resolve('npm', {paths: paths.slice(-1)});
 		} catch (error) {
-			// Error will be thrown if module cannot be found.
-			// We should ignore that error.
+			// If npm module cannot be found, print warning about adding global config to overwrite builtin config
+			console.log(`${chalk.bgYellow.black('\u2009WARN\u2009')} Load npm buildin configs failed, you can use "pnpm config ls" to show buildin configs. And then use "pnpm config --global set <key> <value>" to migrate configs from builtin to global.`);
```

Comment:
You should not print the warning here. You just need to return the warning and then it will be printed by pnpm CLI: https://github.com/pnpm/pnpm/blob/598d6ec8981d2527e57b8c5ca5b2c78df3659bc5/packages/config/src/index.ts#L184

Reply:
Thanks! I didn't know that. I will change it